usaspending_api/api_docs/api_documentation/download/list_downloads.md

```diff
@@ -0,0 +1,50 @@
+## [List Downloads](#list-downloads)
+**Route:** `/api/v2/bulk_download/list_monthly_files/`
+
+**Method:** `POST`
+
+This route lists the monthly files associated with the requested params.
+
+### Request example
+
+```
+{
+    "agency": 425,
+    "fiscal_year": 2015,
+    "type": "contracts"
+}
+```
+
+### Request Parameters Description
+
+* `agency` - specific agency database id. Retrived from the `list_agencies` endpoint.
+* `fiscal_year` - specific fiscal year to pull
+* `type` - specific type of awards/transactions ('contracts' or 'assistance')
```

Comment:
We can remove the word "specific" from each of these descriptions

Reply:
removed